api_tests/btsieve/bitcoin_only/test.ts

```diff
@@ -133,23 +122,31 @@ setTimeout(async function() {
                     expect(res).to.have.status(409);
                 });
 
-                it('btsieve should respond with 204 NO_CONTENT when "getting-or-creating" a new query', async function() {
-                    const newQueryId =
-                        "4BvFBixM4HmhV8AJe5RC8v8csxxhDBscMxwpiK5e";
-                    const newLocation =
-                        "/queries/bitcoin/regtest/transactions/" + newQueryId;
-                    const newQuery = {
-                        to_address: "mndvsV4weBdPFTarHQbg71rCXRy8z79SH5",
-                    };
+                const otherQueryId = "4BvFBixM4HmhV8AJe5RC8v8csxxhDBscMxwpiK5e";
+                const otherLocation =
+                    "/queries/bitcoin/regtest/transactions/" + otherQueryId;
+                const otherQuery = {
+                    to_address: "mndvsV4weBdPFTarHQbg71rCXRy8z79SH5",
+                };
 
+                it('btsieve should respond with 204 NO_CONTENT when "getting-or-creating" a new query', async function() {
```

Comment:
I started a refactor locally, I believe the `PUT` tests should be in a separate file as `PUT` and `POST` are not supposed to be used together.

By having a separate file you ensure there is no mix up nor (un)voluntary usage of side effects.